include/Pothos/Util/ExceptionForErrorCode.hpp

```diff
@@ -0,0 +1,109 @@
+///
+/// \file Util/ErrorCodeHandling.hpp
+///
+/// Mapping sets of error codes to C++ exceptions
+///
+/// \copyright
+/// Copyright (c) 2020 Nicholas Corgan
+/// SPDX-License-Identifier: BSL-1.0
+///
+
+#pragma once
+#include <Pothos/Exception.hpp>
+
+#include <cerrno>
+#include <cmath>
```

Comment:
the cmath header, I cant tell, its not needed, right?

Reply:
It’s used in the errno function to always pass the right code in, since some code returns the negative of the code.